homeassistant/components/light/tellduslive.py

```diff
@@ -0,0 +1,65 @@
+"""
+Support for Tellstick switches using Tellstick Net.
+
+This platform uses the Telldus Live online service.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/light.tellduslive/
+
+"""
+import logging
+
```

Comment:
The only benefit from having it there would be that if a user would add tellduslive platform for covers, it would help guide the user because it would trigger loading the tellduslive component which then would log an error about not being properly configured. Not a deal breaker!